README.md

```diff
@@ -31,6 +31,16 @@ Make sure you have docker desktop installed, WSL2 configured and enough free RAM
 
 Setting up Serge on Kubernetes or docker compose can be found in the wiki: https://github.com/nsarrazin/serge/wiki/Integrating-Serge-in-your-orchestration#kubernetes-example
 
+#### Web3 - Akash Network (AKT) on Cosmos (ATOM)
+
+<a href="https://deploy.cloudmos.io/templates/akash-network-awesome-akash-serge">
+  <img src="https://github.com/88plug/serge/assets/19512127/483ae7f0-b3e4-483b-98d8-108a6a9db9fc" alt="Deploy Serge on Cloudmos">
+</a>
+
+1. Install [Keplr Wallet](https://www.keplr.app/)
+2. Swap ATOM for AKT on https://osmosis.zone/
+3. Connect your Keplr wallet to [https://deploy.cloudmos.io](https://deploy.cloudmos.io/templates/akash-network-awesome-akash-serge) and click Deploy.
```

Comment:
Wouldnt it make more sense to add a deploy.yml on this repo instead of https://github.com/akash-network/awesome-akash/tree/master/serge?

Reply:
That repo is using a very old Dockerfile that doesnt work with serge anymore

Reply:
> 

Akash network only uses the deploy.yaml which references : cryptoandcoffee/akash-serge:1
We try to always publish the source Dockerfile which the Akash compatible image is created with. 

You can also see that here as well on VIEW SDL tab.
https://deploy.cloudmos.io/templates/akash-network-awesome-akash-serge

Reply:
@88plug That is using a very old Dockerfile. Serge got rid of MongoDB and llama.cpp over a month ago

Reply:
@88plug Thoughts?

Reply:
Hi - I will be updating per your request -

Reply:
@88plug Any updates?

Reply:
Doing now...was away over the weekend.

Reply:
https://github.com/akash-network/awesome-akash/pull/386 solves the outstanding issues. We are now on 0.1.3